src/pages/venue.jsx

```diff
@@ -262,6 +262,61 @@ const VanuePage = () => {
                   </p>
                 </Accordion.Detail>
               </Accordion.Item>
+              <Accordion.Item
+                className="border border-gray-800"
+                iconClassName="text-green-500"
+                activeIconClassName="text-white"
+              >
+                <Accordion.Summary>
+                  <p className="typo-intro">
+                    Code of Conduct
+                  </p>
+                </Accordion.Summary>
+                <Accordion.Detail>
+                  <p className="typo-body mb-4">
+                  <List type="bullet" iconClassName="text-white" className="mb-6">
+                    <List.Item className="typo-body">
+                      <span className="mr-2">
+                        Be nice and respectful to others
+                      </span>
+                    </List.Item>
+                    <List.Item className="typo-body">
+                      <span className="mr-2">
+                        Collaborate to create a safe and supportive environment for all participants
+                      </span>
+                    </List.Item>
+                    <List.Item className="typo-body">
+                      <span className="mr-2">
+                        Refrain from harassing speech, content, and other harassing behavior
```

Comment:
maybe say "unwelcome behavior"?

Reply:
Updated in https://github.com/nextflow-io/summit-website/commit/1587f2081cb29b0790b2f670e752b675dca043c5 👍🏻